### ISSUE TITLE
How to use new version of patch?

### ISSUE THREAD

**Issue by ArmandAgopian** [EthanArbuckle/Apollo-CustomApiCredentials#26]
How do I install this new version of the patch? There are only instructions how to create an API key, not how to side load the app itself.

**Comment by Abcmsaj** [comment]
Basically follow the instructions here in the same way: https://www.reddit.com/user/megajigglypuff7I4/comments/1485ygz/how_to_add_personal_api_key_to_apollo/

I have found the ONLY way this works is if you change the bundle identifier when sideloading it... if you use the default one, it will crash on launch

**Comment by jtc2002** [comment]
Ok, I feel like I'm just screwing this up somehow. I installed Ubuntu onto my WSL and have WSL. I was only able to get the Debian based distros to install, but I was able to run the Theos installer without any issues. I believe this is probably my first issue, but I'm not really well versed in Linux.

I am able to open Tweak.m and Makefile and added the line "THEOS_PACKAGE_SCHEME=rootless" and then package it. I then am able to package it and I get a .deb file. I CP that to Windows and then I tweaked an IPA from https://armconverter.com/decryptedappstore/us with the said .deb file. Then, I just sideload that to my iPhone. Unfortunately, the API key does not automatically populate, even when I do not have any other developer API keys. I can see a home page but it does not populate with posts and none of the buttons work.
